prac_08/dynamic_labels.py

```diff
@@ -0,0 +1,24 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import StringProperty
+from kivy.uix.label import Label
+
+class DynamicLabels(App):
+    status_test = StringProperty
```

Comment:
I'm assuming you might have accidentally changed the letter in this variable but change 'test' to 'text' as it sounds like you're trying to test the StringProperty() :)